reporter.go

```diff
@@ -511,20 +511,26 @@ func (r *Reporter) PrintRuleSummary(out io.Writer, report *Report) {
 	fmt.Fprintln(out, "===============================================================================")
 	fmt.Fprintln(out, "Rule Summary:")
 	fmt.Fprintln(out, "===============================================================================")
-	fmt.Fprintln(out, "Client Policy:")
-	if report.WalkBefore == nil {
-		fmt.Fprintln(out, proto.MarshalTextString(report.WalkAfter.Policy))
-	} else {
+
+	if report.WalkBefore != nil {
 		diff := cmp.Diff(report.WalkBefore.Policy, report.WalkAfter.Policy)
 		if diff != "" {
-			fmt.Fprintln(out, "Diff:")
+			fmt.Fprintln(out, "Walks policy diff:")
 			fmt.Fprintln(out, diff)
-			fmt.Fprintln(out, "Before:")
 		}
-		fmt.Fprintln(out, proto.MarshalTextString(report.WalkBefore.Policy))
 	}
-	fmt.Fprintln(out, "Report Config:")
-	fmt.Fprintln(out, proto.MarshalTextString(r.config))
+	if r.Verbose {
+		fmt.Fprintln(out, "Client Policy:")
+		var policy *fspb.Policy
+		if report.WalkBefore == nil {
+			policy = report.WalkAfter.Policy
+		} else {
```

Comment:
An idiomatic thing to do is often to use `if` or `else` part as default value when it improves readability without sacrificing too much:

```go
policy := report.WalkAfter.Policy
if report.WalkBefore != nil {
  policy = report.WalkBefore.Policy
}
proto.MarshalTextString(policy)
```

Up to you though.

Reply:
I don't really like that it looks like the default value should be the `WalkAfter` one when in reality we are only using it if `WalkBefore` is `nil`.

I think I'll go with this anyway since it's shorter and it doesn't matter much.